### PR TITLE
refacture js

### PR DESCRIPTION
**Pull request by Leon910**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,34 +30,46 @@ async function fetchCharacters() {
...
-  pagination.textContent = `${page} / ${maxPage}`;
-}
+console.log(previousClick());
+
+const previousButton = NavButton("previous", previousClick);
+const nextButton = NavButton("next", nextClick);
+
+navigation.append(previousButton, nextButton);
+
+fetchCharacters();
```
Comment (Leon910) [inline index.js]: we could delete one of the fetchCharacters function-call. See also line 43

Comment (Leon910) [reply index.js]: @predrag-babic

File: index.js [hunk 2]
```diff
@@ -28,34 +30,46 @@ async function fetchCharacters() {
...
     });
   } catch (error) {
     console.error("Could not fetch data", error);
   }
-  updatePagination();
 }
+
 fetchCharacters();
+const pagination = NavPagination(page, maxPage);
+navigation.append(pagination);
```
Comment (Leon910) [inline index.js]: can be deleted

File: index.js [hunk 3]
```diff
@@ -28,34 +30,46 @@ async function fetchCharacters() {
...
 
-function updatePagination() {
-  pagination.textContent = `${page} / ${maxPage}`;
-}
+console.log(previousClick());
+
+const previousButton = NavButton("previous", previousClick);
+const nextButton = NavButton("next", nextClick);
+
+navigation.append(previousButton, nextButton);
```
Comment (Leon910) [inline index.js]: can be written as navigation.append(previousButton, pagination, nextButton);

File: index.js (hunk 2 above)
Comment (predrag-babic) [reply index.js]: Yeah i got it. I can do it when i do the searchBar refacturing :)

**Review by predrag-babic — APPROVED** [review]

**Pull request merged**